assets/js/signup.js

```diff
@@ -1,101 +1,92 @@
 $(document).ready(function() {
-  // loadQuestions();
+	// loadQuestions();
 
-  // to be deleted at a later point
-  validateSignupForm();
+	// to be deleted at a later point
+	validateSignupForm();
 
-  // Since non-required fields are "valid" when they are empty, we need an
-  // alternate way to keep labels raised when there is content in their
-  // associated input field
-  $(".vita-form-textfield input").blur(function() {
-    var isBlank = $.trim($(this).val()).length > 0;
-    $label = $(this).siblings(".vita-form-label").toggleClass( "vita-form-label__floating", isBlank );
-  });
+	// Since non-required fields are "valid" when they are empty, we need an
+	// alternate way to keep labels raised when there is content in their
+	// associated input field
+	$(".vita-form-textfield input").blur(function() {
+		var isBlank = $.trim($(this).val()).length > 0;
+		$label = $(this).siblings(".vita-form-label").toggleClass( "vita-form-label__floating", isBlank );
+	});
 
 });
 
 function validateSignupForm() {
-  $("#vitaSignupForm").validate({
-    rules: {
-      "firstName": "required",
-      "lastName": "required",
-      email: {
-        required: true,
-        email: true
-      },
-      phone: {
-        required: true,
-        phoneUS: true
-      }
-    },
-    messages: {
-      email: {
-        required: "We need your email address to confirm your appointment",
-        email: "Your email address must be in the format of name@domain.com"
-      }
-    }
-  });
+	$("#vitaSignupForm").validate({
+		rules: {
+			"firstName": "required",
+			"lastName": "required",
+			email: {
+				required: true,
+				email: true
+			},
+			phone: {
+				required: true,
+				// phoneUS: true
+			}
+		},
+		messages: {
+			email: {
+				required: "We need your email address to confirm your appointment",
+				email: "Your email address must be in the format of name@domain.com"
+			}
+		}
+	});
 }
 
 // Form submission
 $('#vitaSignupForm').submit(function(e) {
-  if (!$(this).valid()) {
-    return false;
-  }
+	// Stop default form submit action
+	e.preventDefault();
```

Comment:
This line redundant. In jQuery, return false; effectively calls e.preventDefault and e.stopPropagation.

See [StackOverflow](https://stackoverflow.com/questions/1357118/event-preventdefault-vs-return-false) for more.